sycl/test-e2e/Sampler/normalized-clampedge-linear-float.cpp

```diff
@@ -1,12 +1,12 @@
 // REQUIRES: aspect-ext_intel_legacy_image
-// UNSUPPORTED: cuda
+//
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
-// XFAIL: target-amd
-// XFAIL-TRACKER: https://github.com/intel/llvm/issues/14732
```

Comment:
I think we should close the XFAIL tracker then: https://github.com/intel/llvm/issues/14732